### PR TITLE
Port several small multiplayer PRs from yuzu

### PR DESCRIPTION
**Pull request by FearlessTobi**
Ports:
https://github.com/yuzu-emu/yuzu/pull/12554
https://github.com/yuzu-emu/yuzu/pull/12852
https://github.com/yuzu-emu/yuzu/pull/12850
https://github.com/yuzu-emu/yuzu/pull/12851

There is one small issue of the contrast being bad when using the dark theme with the multiplayer room list.
Please hold off on merging until I have found a solution with the original PR author.

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/7419)

### CODE REVIEW

**Comment by vitor-k** [comment]
Ctrl+C is already used as a shortcut

**Comment by Calinou** [comment]
> Ctrl+C is already used as a shortcut

The multiplayer shortcuts could be changed to use <kbd>Ctrl + Shift</kbd> instead of just <kbd>Ctrl</kbd> (I'd be OK with doing that on yuzu as well).

File: src/citra_qt/multiplayer/lobby_p.h [hunk 1]
```diff
@@ -188,12 +188,29 @@ class LobbyItemMemberList : public LobbyItem {
     }
 
     QVariant data(int role) const override {
-        if (role != Qt::DisplayRole) {
+        switch (role) {
+        case Qt::DisplayRole: {
+            auto members = data(MemberListRole).toList();
+            return QStringLiteral("%1 / %2 ")
```
Comment (SachinVin) [inline src/citra_qt/multiplayer/lobby_p.h:194]: is the trailing space intended, it wasn't there before.

**Comment by FearlessTobi** [comment]
Ported the fix for the coloring and addressed the review comments.

**Review by Steveice10 — APPROVED** [review]

**Pull request merged**